packages/loader/src/GLTFLoader.ts

```diff
@@ -8,7 +8,7 @@ import { ParserContext } from "./gltf/parser/ParserContext";
 export class GLTFLoader extends Loader<GLTFResource> {
   load(item: LoadItem, resourceManager: ResourceManager): AssetPromise<GLTFResource> {
     const url = item.url;
-    return new AssetPromise((resolve, reject) => {
+    return new AssetPromise((resolve, reject, _, onCancel) => {
```

Comment:
Here don't have type inference and syntax hints.
Maybe should write AssetPromise construct like this:
```ts
constructor(
    executor: (
      resolve: (value?: T | PromiseLike<T>) => void,
      reject?: (reason?: any) => void,
      setProgress?: (progress: number) => void,
      onCancel?: (callback: () => void) => void
    ) => void
  ) 
```